tests/integration/bootstrap-get-images.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+# This script generates "get-images.sh" using Official Images tooling.
+#
+#   ./bootstrap-get-images.sh > get-images.sh
```

Comment:
Might be beneficial to add something like

```bash
#
# This script needs bashbrew. To install the latest version:
# go install github.com/docker-library/bashbrew/cmd/bashbrew@latest
```

Reply:
Sure, added a note about `bashbrew` here :+1: